components/src/preact/mutationComparison/mutation-comparison.tsx

```diff
@@ -190,7 +191,32 @@ const MutationComparisonInfo: FunctionComponent<MutationComparisonInfoProps> = (
     return (
         <Info>
             <InfoHeadline1>Info for mutation comparison</InfoHeadline1>
-            <InfoParagraph>TODO: https://github.com/GenSpectrum/dashboard-components/issues/465</InfoParagraph>
+            <InfoParagraph>
+                This shows <SubstitutionsLink /> and <DeletionsLink /> of several variants. It shows mutations where the
```

Comment:
```suggestion
               This displays <SubstitutionsLink /> and <DeletionsLink /> of several variants. It shows mutations where the proportion for any given variant falls within the range you can select in the component's toolbar.
```
GPT suggestion, which I like :)